Readme.md

```diff
@@ -50,6 +51,17 @@ $ xbuild
 
 Requires NixOS 16.09+.
 
+#### MacOS
+
+You'll need [Mono][mono] and [NuGet][nuget] installed.
+
+Either use [JetBrains Rider][jetbrains-rider] to open and build `MoonBaseBuilder.sln`
+file, then invoke the following commands console:
```

Comment:
"Currently, we recommend to build the project using JetBrains Rider. Open and build `MoonBaseBuilder.sln` file, then invoke the following command in the terminal:"